src/isomorphic/classic/class/__tests__/ReactClass-test.js

```diff
@@ -189,9 +190,8 @@ describe('ReactClass-spec', function() {
       'mean shouldComponentUpdate()? The name is phrased as a question ' +
       'because the function is expected to return a value.'
     );
-
-    <NamedComponent />; // Shut up lint
```

Comment:
For this one, we could just do `displayName: 'NamedComponent',` inside the class spec and drop the var assignment.